rflx/cli.py

```diff
@@ -190,6 +191,9 @@ def main(argv: List[str]) -> Union[int, str]:  # pylint: disable = too-many-stat
     )
     parser_validate.set_defaults(func=validate)
 
+    parser_setup = subparsers.add_parser("setup", help="set up RecordFlux")
```

Comment:
I think this is misleading. When I'm able to run this command RecordFlux must already be set up. People who try to set up RecordFlux may wonder why it doesn't actually set up RecordFlux and people who try to find the IDE integration may not find this command because it doesn't say it sets up the IDE.

Reply:
Fair enough. I don't want to make to specific to GNAT Studio, though. And @treiher's original idea is pretty bulky IMHO. How about "setup_ide"?

Reply:
I think `setup_ide` or `ide_setup` is a good choice. Maybe it's worth to make the help comment a bit more extensive and add the supported IDEs. While I see the point that it shouldn't be GNAT Studio specific, we should at least tell the supported IDEs in the help.